CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 1.1.2-wip
```

Comment:
New features should get a minor version bump - we will need to update the pubspec.yaml version as well.

```suggestion
## 1.2.0-wip
```